### PR TITLE
Stabilized light pink extract makes you a pacifist while active

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

The stabilized light pink extract (the ambulance one) now also makes you a pacifist while it affects you.

## Why It's Good For The Game

Pretty much all the stabilized extracts are absurd in some way so someone's gotta start to bring them in at some point. (Or I could just remove them all.)

## Changelog
:cl: Melbert
balance: The stabilized light pink extract now makes you a pacifist while active.
/:cl:

### CODE REVIEW

File: code/modules/research/xenobiology/crossbreeding/_status_effects.dm [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#define STABILIZED_LIGHT_PINK_TRAIT "stabilized_light_pink"
```
Comment (Timberpoes) [inline code/modules/research/xenobiology/crossbreeding/_status_effects.dm:1]: Could ya cram this somewhere in code/__DEFINES/traits.dm pretty please? It keeps all the trait sources in one tidy place for quicker/easier reference and allows future coders to reference this source from outside of the code in the future.

**Review by Timberpoes — CHANGES_REQUESTED** [review]
Just one minor request. Concept approved, a good way to start hitting some of Xenobio's power. Thematically fits with the description text.

**Comment by moocowswag** [comment]
I kind of like this change because it doesnt prevent xenobiologists from firing off extracts and doing things that are (more limited) but more related to their job and being creative.

**Review by Timberpoes — APPROVED** [review]

**Comment by ATH1909** [comment]
isn't pacifism a pink extract thing, not a light pink extract thing?

**Comment by D4C-420** [comment]
> isn't pacifism a pink extract thing, not a light pink extract thing?

Kind of? The majority of light pink cross breeds made and listed in the design doc apply pax or force them to be pacifist (best example I think would be charged light pink which forces pacifism trauma on a person) so its not unprecedented but they both kind of do it. 

On the changes I mean xenobio does need balance and for the stable extracts I guess having downsides works for balance.

**Comment by tommysalami3** [comment]
But HOW will I powergame as sec NOW????

**Comment by moocowswag** [comment]
> But HOW will I powergame as sec NOW????

This change wont hurt security, if anything its an indirect buff because they can easily make the most use of it despite pacifism by using disablers and batons.

**Pull request merged**